### PR TITLE
Feat: Avatar and Buttons component included

### PR DESCRIPTION
**Pull request by gopalbbharadva**
### This is a PR for **Avatar** and **Button** component.

- I made 4 variants for Avatar and 5 variants for Buttons.
- Styling and colors to show them is temporary. I will change them in actual documentation.

**Avatar Output:**
image

**Button Output:**
image

### CODE REVIEW

**Comment by nshanbhag20001412** [comment]
looks good to me. If possible keep a live preview or update the links of the navbar so that one can see the preview using sandbox

**Review by nshanbhag20001412 — APPROVED** [review]

**Review by Sen-442b — APPROVED** [review]
Looks clean, you can also add bit border-radius and box-shadow in your button component

File: Components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+.avatar-letter {
+  border: 1px solid var(--Color_Black);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--Primary_Color);
+}
+.first {
```
Comment (msdeshmukh009) [inline Components/Avatar/avatar.css:47]: Maybe you can use class names like font-size-lg, font-size-sm, font-size-md, etc. which will convey more meaning. Other things look fine to me.

**Comment by gopalbbharadva** [comment]
> Looks clean, you can also add bit border-radius and box-shadow in your button component

Ok

**Review by msdeshmukh009 — APPROVED** [review]

File: Components/Avatar/avatar.css (hunk 1 above)
Comment (gopalbbharadva) [reply Components/Avatar/avatar.css:47]: ya I can do that also. thanks i will update

**Pull request merged**